### PR TITLE
basic genus node

### PR DESCRIPTION
**Pull request by jjthomas**

### CODE REVIEW

**Comment by jjthomas** [comment]
I'm getting errors like the following (they don't terminate genus though, and it seems to complete successfully anyway), so I may be failing to source some tech info:

```
Error   : Undefined pin layer detected. [PHYS-148] [set_attribute]
        : The layer 'PW' referenced in pin 'VBB' in macro 'INVD2BWP16P90ULVT' is not found in the database.
```

**Comment by alexcarsello** [comment]
Could you provide a pointer to a run that used this node?

**Comment by jjthomas** [comment]
`/sim/jjthomas/garnet/build_Tile_MemCore_pwr_all` on r7arm-aha. This run uses all 3 scenarios.

File: steps/cadence-genus-synthesis/configure.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+#=========================================================================
+# Cadence Genus Synthesis
+#=========================================================================
+# Author : Christopher Torng, Yanghui Ou
```
Comment (ctorng) [inline steps/cadence-genus-synthesis/configure.yml:4]: This is the wrong author

**Comment by ctorng** [comment]
Is this node ready to merge to master?

**Comment by alexcarsello** [comment]
The log file from that run is a bit concerning... tons of errors about undefined layers and lib cells being marked as "avoid"

**Comment by ctorng** [comment]
Is the genus script basically the same as the one used in tapeout_16? We can cross-check for quality against the old Jade builds right?

**Comment by alexcarsello** [comment]
It's similar yes. I found an old log, and it doesn't have all of those errors.

**Comment by jjthomas** [comment]
It probably has something to do with some of the extra libs or libs from the adk that are being sourced ... not sure which one exactly is the issue.

**Comment by alexcarsello** [comment]
Any more info on these errors?

**Comment by alexcarsello** [comment]
The errors are gone with this last change, so I think this is ready to merge now.

**Pull request merged**